src/app/static/src/app/components/Stepper.vue

```diff
@@ -159,9 +160,13 @@
                 }
                 return !this.isComplete(activeStep)
             },
+            clearReviewInputWarnings: function() {
+                this.clearSurveyAndProgramWarnings()
+                this.clearGenericChartWarnings()
+            },
             clearWarnings(){
                 const mutationMethods: { [key: number]: () => void; } = {
-                    2: this.clearReviewInputsWarnings,
+                    2: this.clearReviewInputWarnings,
```

Comment:
I think `clearReviewInputsWarnings` is probably a better name since `Review Inputs` is the name of the step